test/Interop/Cxx/stdlib/overlay.swift

```diff
@@ -0,0 +1,21 @@
+// RUN: %target-run-simple-swift(-I %S/Inputs -Xfrontend -enable-experimental-cxx-interop)
```

Comment:
Nit: maybe it makes more sense to have this be `Cxx/stdlib/overlay/string.swift` or `Cxx/overlay/string.swift`. WDYT?

Reply:
Yeap, let's make it `Cxx/stdlib/overlay/string.swift`